clients/tests/src/main/java/org/projectnessie/client/tests/AbstractSparkTest.java

```diff
@@ -67,8 +64,6 @@ protected void create() throws IOException {
         .set("spark.sql.catalog.nessie", "org.apache.iceberg.spark.SparkCatalog");
     spark = SparkSession.builder().master("local[2]").config(conf).getOrCreate();
     spark.sparkContext().setLogLevel("WARN");
-
-    nessieClient = NessieClient.builder().withUri(url).build();
   }
 
   @AfterAll
```

Comment:
Can you change this to `@AfterEach` as well? It feels weird to initialize a SparkSession for every test but only clean up the last one. Can be a follow-up though.

Reply:
yeah should probably be a follow-up